WooCommerce/Classes/ViewRelated/Hub Menu/HubMenuViewModel.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+import UIKit
+import SwiftUI
+
+/// View model for `HubMenu`.
+///
+final class HubMenuViewModel: ObservableObject {
+
+    let siteID: Int64
+
+    let storeTitle = ServiceLocator.stores.sessionManager.defaultSite?.name ?? Localization.myStore
+    var storeURL: URL {
+        guard let urlString = ServiceLocator.stores.sessionManager.defaultSite?.url, let url = URL(string: urlString) else {
+            return WooConstants.URLs.blog.asURL()
+        }
+
+        return url
+    }
+
+    /// Child items
+    ///
+    @Published private(set) var menuElements: [Menu] = []
+
+    init(siteID: Int64) {
+        self.siteID = siteID
+        menuElements = [.woocommerceAdmin, .viewStore, .reviews]
+    }
+}
+
+extension HubMenuViewModel {
+    enum Menu: CaseIterable {
+        case woocommerceAdmin
+        case viewStore
+        case reviews
+
+        var title: String {
+            switch self {
+            case .woocommerceAdmin:
+                return Localization.woocommerceAdmin
+            case .viewStore:
+                return Localization.viewStore
+            case .reviews:
+                return Localization.reviews
+            }
+        }
+
+        var icon: UIImage {
+            switch self {
+            case .woocommerceAdmin:
+                return .wordPressLogoImage.imageWithTintColor(.blue) ?? .wordPressLogoImage
+            case .viewStore:
+                return .storeImage.imageWithTintColor(.accent) ?? .storeImage
+            case .reviews:
+                return .starImage(size: 24.0).imageWithTintColor(.primary) ?? .starOutlineImage()
```

Comment:
super nit: is it intentional to have a different default image instead of: 
```suggestion
                return .starImage(size: 24.0).imageWithTintColor(.primary) ?? .starImage()
```